README.md

```diff
@@ -1,27 +1,30 @@
-# Alpha Status
+# F5 Networks Helm Charts
 
-This project is presently in alpha status and chart availabiliy in the helm repos below is limited and may be unstable.
+This repository contains [helm](https://docs.helm.sh/using_helm/#using) charts for use with some [F5 Networks](https://f5.com/) products and services within a [Kubernetes](https://kubernetes.io/) or [OpenShift](https://www.openshift.com/) environment.
 
-# F5 Networks Helm Charts
+**Note** that charts may require access to `kube-system` namespace and/or cluster wide permissions to function as expected, so be sure to install/configure helm/tiller appropriately.
+
+## Stable Charts
 
-To use the charts in this repository to deploy the [F5 k8s-bigip-ctlr](https://github.com/F5Networks/k8s-bigip-ctlr) using [Ingress](https://kubernetes.io/docs/concepts/services-networking/ingress/) or [ConfigMap](https://kubernetes.io/docs/tasks/configure-pod-container/configure-pod-configmap/):
+To add the stable repo to helm:
 
 ```
 helm repo add f5-stable https://f5networks.github.io/charts/stable
-helm install f5-stable/f5-bigip-ctlr
 ```
 
-To access additional charts in beta:
+These charts are tested and documented:
```

Comment:
Stolen from the openstack heat project repos, feel free to amend or disregard as appropriate:
```
The stable directory contains charts that are created and tested by F5 Networks. These templates are supported by F5 (see [SUPPORT](./SUPPORT) for details).
```
Also, add the SUPPORT file if appropriate (maybe check with Ashok?) - https://github.com/F5Networks/k8s-bigip-ctlr/blob/master/SUPPORT.md

Reply:
Used part of the language and will ask Ashok about support.

Reply:
Adding the support reference per offline conversation with @amudukutore